### PR TITLE
add _get_raw_bliss_lexicon

### PR DESCRIPTION
**Pull request by JackTemaki**
I did some code restructuring in preparation for an updated Librispeech Lexicon pipeline.

After building some setups it seems it was a bad idea to first merge the static symbols and then do the G2P, it would save a lot of job cases when this would be the other way around. This PR does not introduce this yet (a second get_g2p_augmented_lexicon), but just does some initial reformatting.

### CODE REVIEW

**Comment by JackTemaki** [comment]
@christophmluscher @Atticus1806 

Ping ;)

File: common/datasets/librispeech.py [hunk 1]
```diff
@@ -354,6 +354,50 @@ def get_special_lemma_lexicon(add_unknown_phoneme_and_mapping=True):
     return lex
 
 
+@lru_cache()
+def _get_raw_bliss_lexicon(
+    use_stress_marker,
```
Comment (Atticus1806) [inline common/datasets/librispeech.py:359]: Is there a reasonable default for this bool? Or is this intentionally not set? `get_bliss_lexicon` has `False` as default.

**Review by Atticus1806 — COMMENTED** [review]
Looks good, besides this default everything is fine I think.

>Ping ;)
Pong.

**Review by christophmluscher — APPROVED** [review]

File: common/datasets/librispeech.py (hunk 1 above)
Comment (christophmluscher) [reply common/datasets/librispeech.py:359]: If you want to use this private function, you should think about what you want to do. so I think there is no reasonable default, IMHO

**Pull request merged**